tower-http/src/lib.rs

```diff
@@ -211,7 +210,7 @@
     nonstandard_style,
     missing_docs
 )]
-#![deny(unreachable_pub, private_in_public)]
```

Comment:
I think this lint got replaced by smaller individual lints? Probably worth enabling those / some of them.